README.md

```diff
@@ -1,2 +1,4 @@
-# triton-touchbase
-Touchbase stack designed for container-native deployment on Joyent's Triton platform
+triton-elasticsearch
```

Comment:
Elastic?

Separately, I don't know if it's significant that the license in this repo is different from [the license in the Touchbase repo](https://github.com/couchbaselabs/touchbase/blob/master/LICENSE.md).